src/dependencytracker.js

```diff
@@ -252,6 +255,30 @@ class DependencyTracker {
         }
       },
     });
+
+    let task = await this.Task.load({taskId: taskId});
```

Comment:
I think the most common call site for this is from dependencyResolver which doesn't have taskGroupId or load a Task before calling dependencyTracker.

That said, the places that call `putResolvedMessage` both have task loaded and ready, so we can just add it to the message in the azure queue, right? What's the migration path for that look like? Is there a scheme associated with it and such or do we just need to be able to handle both message formats for a bit and all will work out ok?